Client/game_sa/CPedSA.h

```diff
@@ -408,4 +408,6 @@ class CPedSA : public virtual CPed, public virtual CPhysicalSA
     void*                   GetPedNodeInterface(std::int32_t nodeId) { return reinterpret_cast<CPedSAInterface*>(m_pInterface)->pedNodes[nodeId]; }
     std::unique_ptr<CPedIK> GetPedIK() { return std::make_unique<CPedIKSA>(GetPedIKInterface()); }
     static void             StaticSetHooks();
+
+    void GetAttachedSatchels(std::vector<SSatchelsData> &satchelsList);
```

Comment:
why not return a vector?

Reply:
It is better to avoid copying the entire vector. In addition, let's keep it consistent with the project structure. Look at functions like GetPosition, GetMoveSpeed ​​etc.